test/e2e/framework/pods.go

```diff
@@ -37,7 +37,7 @@ import (
 // node e2e test.
 var ImageWhiteList sets.String
 
-// Convenience method for getting a pod client interface in the framework's namespace,
+// Convenient method for getting a pod client interface in the framework's namespace,
```

Comment:
I think the previous form "[Convenience method](http://wiki.c2.com/?ConvenienceMethods)" was better, but that's just one opinion :)